scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -153,12 +153,24 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
                                gene_panels=['hpo']) }}">
                Clinical HPO SNV variants
              </a>
+            <a class="btn btn-secondary btn-sm text-white"
+              href="{{ url_for('variants.cancer_sv_variants', institute_id=institute._id,
+                               case_name=case.display_name, variant_type='clinical',
+                               gene_panels=['hpo']) }}">
+               Clinical HPO SV variants
+             </a>
```

Comment:
This looks like this:

<img width="609" alt="image" src="https://user-images.githubusercontent.com/28093618/188794771-523e6e76-7aae-42fc-a94c-142f2518560a.png">

Just an idea so feel free to accept it or not: since there is some space on the left of the SNVs button, perhaps both buttons could fit on the same line?

Reply:
Hihi, yes, they actually do, but there just isn't enough space on your portable display:
![Screenshot 2022-09-07 at 09 09 01](https://user-images.githubusercontent.com/758570/188812261-fe5d4dbe-3aa5-404d-89f4-215bc82beecf.png)

I'll try with a button group and see if it looks intuitive!

Reply:
Move both and also the red one above a bit on the left?

Reply:
How about this, with a btn-group. Its more compact and probably still conveys the same message:
![Screenshot 2022-09-07 at 09 19 08](https://user-images.githubusercontent.com/758570/188814368-11166271-c745-480e-b9ea-3edbb46a44ea.png)

We could justify both to the right as well perhaps.

Reply:
The grey buttons look a bit unbalanced I think.

Reply:
![Screenshot 2022-09-07 at 09 38 58](https://user-images.githubusercontent.com/758570/188818418-96e34404-03d2-4b9d-86d1-bc3cbc8b1365.png)
Hm, I dont know - I find it a bit harder to read when the button texts above one another don't start at the same column.

Reply:
Ok and a button with label "Clinical HPO SVs" doesn't fit the space?

Reply:
That all depends on the screen size.. But, in a group it will swell downwards instead of wrapping over to the next row, so, I'm putting the text back and leaving the left-justified button group.

Reply:
Like this:
![Screenshot 2022-09-07 at 09 47 03](https://user-images.githubusercontent.com/758570/188820379-924b084f-e63b-4f64-a3f5-771f764ef7ef.png)
![Screenshot 2022-09-07 at 09 47 09](https://user-images.githubusercontent.com/758570/188820387-77dc7bb1-b119-4971-b9ec-ebe3ea40cd40.png)

It will do I think.

Reply:
Looks good I think!